pkg/github/tools.go

```diff
@@ -82,6 +86,7 @@ func InitToolsets(passedToolsets []string, readOnly bool, getClient GetClientFn,
 	tsg.AddToolset(users)
 	tsg.AddToolset(pullRequests)
 	tsg.AddToolset(codeSecurity)
+	tsg.AddToolset(actions)
```

Comment:
@SamMorrowDrums let me know if I added this correctly

Looking through #188, it seems like we don't want some toolsets enabled by deafult but it's not clear to me what enables that.

> Default Enabled Features
> By default, the following features are enabled:
>...

As far as I can tell, the default is "all"

https://github.com/github/github-mcp-server/blob/614f2267249ee9e4aeb8696d29caa93f3752196a/pkg/github/tools.go#L14